OpenOversight/app/utils.py

```diff
@@ -62,6 +62,14 @@ def dept_choices():
     return db.session.query(Department).all()
 
 
+def year_choices():
+    years = [year[0] for year in db.session.query(cast(func.extract('year', Officer.employment_date), Integer)).filter(Officer.employment_date.isnot(None)).order_by(func.extract('year', Officer.employment_date))]
+    years_deduped = list(set(years))
```

Comment:
I realize this isn't a super efficient way to dedupe the years.  Please let me know if anyone can think of a more performant approach.

Reply:
Could we just hardcode the years to be from like the year we founded OO or something (open to discussion how far back to go), to the current year?

Reply:
Even though the current code isn't ideal, I prefer it to hardcoding a list of years.  I think hardcoding is brittle and vulnerable to breakage.

Reply:
I am not sure I understand what exactly we want to get here. I think whatever years we offer to select should be an uninterrupted range. If it was 1990, 1995, 2003 and 2010 a user would probably be somewhat confused. If you want 2005, do you select 2003 or 2010? I would either hardcode the start year (maybe 2014 or something, I don't think we have much data from before that) and go until current year or take the lowest that's in the database and offer every year from that year on until current year

Reply:
My goal with `year_choices()` is to display every year for which a department has roster information.  
> take the lowest that's in the database and offer every year from that year on until current year

I went for this approach.